src/collection-preferences/content-display/sortable-item.scss

```diff
@@ -42,9 +42,7 @@ $border-radius: awsui.$border-radius-item;
 .sortable-item-content {
   border-top: awsui.$border-divider-list-width solid awsui.$color-border-divider-default;
   display: flex;
-  flex-wrap: nowrap;
-  justify-content: space-between;
```

Comment:
This line was not necessary because one of the children has `flex-grow: 1`, therefore there is no space left to distribute between them.